lib/mumukit/auth/permissions_persistence.rb

```diff
@@ -0,0 +1,27 @@
+module Mumukit::Auth
+  class PermissionsPersistence
+    def self.from_env
```

Comment:
Why have you named it `Mumukit::Auth::PermissionsPersistence` instead of `Mumukit::Auth::Permissions::DaybreakPersistence` or something like that?